src/core/src/bootstrap/Constants.py

```diff
@@ -98,6 +98,27 @@ class ConfigSettings(EnumBackport):
     # patch versions for healthstore when there is no maintenance run id
     PATCH_VERSION_UNKNOWN = "UNKNOWN"
 
+    # Strings used in perf logs
+    class LogStrings:
```

Comment:
LogStrings seems generic and doesn't convey the exact use for this. How about PerfLogTrackerParams? Or open to any suggestions you have. The name should identify what it is used for

Reply:
Renamed to PerfLogTrackerParams